TileFormats/VectorData/README.md

```diff
@@ -1,19 +1,198 @@
 # Vector Data
 
-* [Draft spec](https://github.com/AnalyticalGraphicsInc/3d-tiles/pull/124/files)
-* GitHub issue, [#25](https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/25)
+## Contributors
 
-## Notes
+* Dan Bagnell, [@bagnell](https://github.com/bagnell)
+* Rob Taglang, [@lasalvavida](https://github.com/lasalvavida)
+* Patrick Cozzi, [@pjcozzi](https://twitter.com/pjcozzi)
 
-**Use cases**: Traditional geospatial features: points, polylines, and polygons.  Replacing KML.
+## Overview
 
-**Format**
-* Combination of binary (for positions, normals, etc.) and JSON (for labels, other metadata, etc.).
-* Concise representations for Cesium's [set of geometries](http://cesiumjs.org/2013/11/04/Geometry-and-Appearances/), including extrusions, and billboards and labels.
-   * Need to carefully select the representation for the best trade-off between conciseness and runtime processing.  For example, polygons will likely be pre-triangulated since it only adds indices to the payload, but will be subdivided at runtime since subdivision is fast and increases the vertex payload significantly.
-   * RTC positions for high precision-rendering.
-   * Context-aware compression.
-   * Bounding volume may need to be adjusted at runtime for terrain clamping.
-* Metadata for cracking, morphing, and perhaps label declutter.
+The _Vector_ tile format allows streaming of vector datasets including points, polylines, and polygons.  Points can be represented with a combination of billboard, label, and point graphics primitives.
 
-Could also name this a `Geometry` tile.
+Each point, poyline, and polygon is a _feature_ in the 3D Tiles specification language. 
+
+## Layout
+
+A tile is composed of two sections: a header immediately followed by a body.
+
+**Figure1**: Vector tile layout.
+![layout](figures/layout.jpg)
```

Comment:
Also, for general meshes, we would should prefer to quantize like glTF.

When I was experimenting with different formats, the current form compressed better. But since we want to add general meshes soon, is it better to switch back to the same quantization that glTF uses for consistency? Then we could make the buffer glTF-like as well.